### PR TITLE
[9.0] [ADD] New module l10n_be_partner_identification

### PR DESCRIPTION
**Pull request by lmignon**
This one depends of https://github.com/OCA/partner-contact/pull/255

### CODE REVIEW

File: l10n_be_partner_identification/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Belgium Partner Identification Numbers
+======================================
+
+This addons extentds Partner Identification Numbers to provide:
```
Comment (laetitia-gangloff) [inline l10n_be_partner_identification/README.rst]: extends instead of extentds

File: l10n_be_partner_identification/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================================
+Belgium Partner Identification Numbers
+======================================
+
+This addons extends *Partner Identification Numbers* to provide:
+* a Belgium national registration number category. The category validate the number according the the rules
```
Comment (elicoidal) [inline l10n_be_partner_identification/README.rst]: Dont make a list if there is only one item

File: l10n_be_partner_identification/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (elicoidal) [inline l10n_be_partner_identification/__init__.py:2]: add your copyright for consistency

File: l10n_be_partner_identification/models/res_partner_id_category.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+class ResPartnerIdCategory(models.Model):
+    _inherit = "res.partner.id_category"
+
+    @api.model
+    def validate_l10n_be_national_registry_number(self, id_number):
+        """Method called from the validation_code to validate a belgium
+        `Wikipedia
+         <https://fr.wikipedia.org/wiki/Num%C3%A9ro_de_registre_national>`_
+         The method doesn't support number for persons more than 100 years old.
```
Comment (elicoidal) [inline l10n_be_partner_identification/models/res_partner_id_category.py:18]: that is a major drawback ;)

File: l10n_be_partner_identification/tests/test_l10n_be_partner_identification.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#  ©  2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2._psycopg import IntegrityError
```
Comment (elicoidal) [inline l10n_be_partner_identification/tests/test_l10n_be_partner_identification.py]: Should you protect the import with Try...Except?

File: l10n_be_partner_identification/__init__.py (hunk 3 above)
Comment (lmignon) [inline l10n_be_partner_identification/__init__.py:2]: IMO copyright in **init** is useless

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by ThomasBinsfeld — APPROVED** [review]
Code review

**Pull request closed without merging**